Documentation/install/upgrade.rst

```diff
@@ -686,37 +686,37 @@ Upgrading from >=1.4.0 to 1.5.y
 
         .. parsed-literal::
 
-          $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/1.10/cilium-pre-flight-with-rm-svc-v2.yaml
+          $ kubectl apply -f https://raw.githubusercontent.com/cilium/cilium/v1.5.12/examples/kubernetes/1.10/cilium-pre-flight-with-rm-svc-v2.yaml
```

Comment:
I think we can just go ahead and pin them all to the latest v1.5 docs.

We are unlikely to make any further changes to these on that branch at this stage.

```suggestion
          $ kubectl apply -f https://raw.githubusercontent.com/cilium/cilium/v1.5/examples/kubernetes/1.10/cilium-pre-flight-with-rm-svc-v2.yaml
```

Reply:
done!